browser.go

```diff
@@ -438,11 +453,10 @@ func (w *Window) SetFramebufferSizeCallback(cbfun FramebufferSizeCallback) (prev
 }
 
 func (w *Window) GetSize() (width, height int) {
-	// TODO: See if dpi adjustments need to be made.
-	fmt.Println("Window.GetSize:", w.canvas.GetBoundingClientRect().Width, w.canvas.GetBoundingClientRect().Height,
-		"->", int(w.canvas.GetBoundingClientRect().Width), int(w.canvas.GetBoundingClientRect().Height))
+	width = int(w.canvas.GetBoundingClientRect().Width*w.devicePixelRatio + 0.5)
```

Comment:
What is 0.5 here?

Reply:
It is a way to do rounding from float to int to the upper boundary when above .5 and the below boundary when below .5 for any number.